andrewinsoul/Fast-Food-Fast
# implement the functionality that allows a user get all orders

### WHAT DOES THIS PR DO?
Have the get all orders endpoint working in Fast Food REST API

### DESCRIPTION OF TASK TO BE COMPLETED
- Set up express server
- Have GET /orders working

### HOW SHOULD THIS BE MANUALLY TESTED?
After cloning the repo, cd into it and RUN npm start, using postman, test the endpoint GET /orders
It should return status 200 with an empty array as no orders are yet added.

###WHAT ARE THE RELEVANT PIVOTAL TRACKER STORIES?
#160539515
